aerleon/lib/junipersrx.py

```diff
@@ -529,8 +537,8 @@ def _TranslatePolicy(self, pol: policy.Policy, exp_info: int):
                 # SRX policies are controlled by addresses that are used within, so
                 # policy can be at the same time inet and inet6.
                 if self._GLOBAL_ADDR_BOOK in self.addr_book_type:
-                    for zone in self.addressbook.addressbook:
-                        for _, ips in sorted(self.addressbook.addressbook[zone].items()):
+                    for zone in self.addressbook._ip_addressbook:
+                        for _, ips in sorted(self.addressbook._ip_addressbook[zone].items()):
```

Comment:
Should this code use one of the new addressbook methods?

Reply:
naaaa